products/app-lifecycle-mgmt/{{cookiecutter.product_name}}/config/solutionsv2/shared/{{cookiecutter.segment_name}}/solution.json

```diff
@@ -76,10 +76,10 @@
                                             }
                                         },
                                         "Links" : {
-                                            "slave-codeontap" : {
+                                            "slave-hamlet" : {
```

Comment:
We should also rename slave to agent

Reply:
Doneski https://github.com/hamlet-io/executor-cookiecutter/pull/8/commits/471f02441880af09fa5c2b8588f1cd63b112831f